DFC.App.ExploreCareers/Startup.cs

```diff
@@ -36,22 +48,82 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             else
             {
                 app.UseExceptionHandler("/Home/Error");
-                // The default HSTS value is 30 days. You may want to change this for production scenarios, see https://aka.ms/aspnetcore-hsts.
                 app.UseHsts();
             }
+
+            app.UseCors(_corsPolicy);
             app.UseHttpsRedirection();
             app.UseStaticFiles();
-
             app.UseRouting();
+            app.UseEndpoints(endpoints =>
+            {
+                endpoints.MapRazorPages();
 
-            app.UseAuthorization();
+                // add the default route
+                endpoints.MapControllerRoute("default", "{controller=Health}/{action=Ping}");
+            });
+            mapper?.ConfigurationProvider.AssertConfigurationIsValid();
+        }
 
-            app.UseEndpoints(endpoints =>
+        public void ConfigureServices(IServiceCollection services)
+        {
+            var cosmosDbConnectionContentPages = configuration.GetSection(CosmosDbContentPagesConfigAppSettings).Get<CosmosDbConnection>();
+            services.AddDocumentServices<JobCategory>(cosmosDbConnectionContentPages, env.IsDevelopment());
+
+            services.AddApplicationInsightsTelemetry();
+            services.AddHttpContextAccessor();
+            services.AddTransient<IEventMessageService<JobCategory>, EventMessageService<JobCategory>>();
+            services.AddAutoMapper(typeof(Startup).Assembly);
+            services.AddHostedServiceTelemetryWrapper();
+            services.AddSubscriptionBackgroundService(configuration);
+            services.AddTransient<IDocumentService<JobCategory>, DocumentService<JobCategory>>();

```

Comment:
services.AddTransient<IDocumentService<JobCategory>, DocumentService<JobCategory>>();

Is not needed as this is done in the Comsos nuget as part of AddDocumentServices